src/components/Auth/api.test.ts

```diff
@@ -0,0 +1,144 @@
+import AuthApi from './api';
+import { AuthUser } from './types';
+
+describe('API', () => {
+  const hostAndPort = 'piroj.ok';
+  const projectId = 'pelmen-the-project';
+
+  function setup(result: unknown) {
```

Comment:
nit: change signature to `setup({mockFetchResult: []})` so tests below are more clear.